apps/site/test/site_web/views/teaser_view_test.exs

```diff
@@ -96,4 +120,29 @@ defmodule SiteWeb.TeaserViewTest do
       assert display_location(location) == "Quincy, MA"
     end
   end
+
+  describe "handle_fields/2" do
+    test "returns the proper default fields for display by content type" do
+      cms_default_value = []
+      elixir_default_value = nil
```

Comment:
Is this again referring to the "`cms` app vs. other apps" boundary, or is it something else?